Plant-for-the-Planet-org/planet-donations
# implemented new payment config

Changes made:
- New payment config implementation

What to test-
1. Test donation with all purposes and payment methods
2. Check the amount, quantity and frequency of API requests
3. Check direct donations also

Few Project Ids to test:
- Membership: proj_LOxkf5GYI054Fi0HcEUF3dKu
- Funding: proj_bFH0BU0Qw02RuetpQlLOMVYX
- Bouquet: proj_6x3GTD5cMRv0OeQAiIlJZ0Au
- Trees: proj_jIg2aW8I6RkLHwLcEZodh5Wq
- Conservation: proj_EZQ1nnevs6MoIjIzjlAFjU6M
